README.md

```diff
@@ -93,8 +101,15 @@ The configuration file can contain multiple areas to request tiles for by adding
 
 The parameters in the file are as follows:
 * **description** - A string containing a description of the area
+* **serverprotocol** - The protocol to be used for server connection ("http" or "https").
```

Comment:
Could you please make the serverprotocol optional and default it to https?  Or should it default to http for backwards compatibility?

Reply:
Done.